UnityProject/Assets/Scripts/Tool Construction Deconstruction/ObjectState.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public enum ObjectState 
```

Comment:
As we have neglected namespaces for so long things like this will get confusing. Put a summary comment at the top

Reply:
Still requires summary comments